x-pack/test/security_solution_cypress/cypress/e2e/entity_analytics/host_details/risk_tab.cy.ts

```diff
@@ -17,39 +15,7 @@ import { navigateToHostRiskDetailTab } from '../../../tasks/host_risk';
 import { deleteAlertsAndRules } from '../../../tasks/api_calls/common';
 
 describe('risk tab', { tags: ['@ess'] }, () => {
-  describe('with legacy risk score', () => {
-    beforeEach(() => {
-      cy.task('esArchiverLoad', { archiveName: 'risk_hosts' });
-      login();
-      deleteRiskEngineConfiguration();
-    });
-
-    after(() => {
-      cy.task('esArchiverUnload', { archiveName: 'risk_hosts' });
-    });
-
-    it('renders risk tab', () => {
-      visitHostDetailsPage('siem-kibana');
-      navigateToHostRiskDetailTab();
-
-      cy.get('[data-test-subj="topRiskScoreContributors"]')
-        .find(TABLE_ROWS)
-        .within(() => {
-          cy.get(TABLE_CELL).contains('Unusual Linux Username');
-        });
-    });
-
-    it('shows risk information overlay when button is clicked', () => {
-      visitHostDetailsPage('siem-kibana');
-      navigateToHostRiskDetailTab();
-
-      openRiskInformationFlyout();
-
-      cy.get(RISK_INFORMATION_FLYOUT_HEADER).contains('Entity Risk Analytics');
-    });
-  });
-
-  describe('with new risk score', { tags: ['@serverless'] }, () => {
+  describe('with risk score', { tags: ['@serverless'] }, () => {
```

Comment:
Having 2 nested describes now is not truly needed. It might be better to have only one and move the serverless tag to the above.

Reply:
[remove describe block](https://github.com/elastic/kibana/pull/201810/commits/dd58dd96d9d6f1728495f210150321de92382230)